src/test/kotlin/camp/nextstep/edu/racingcar/interfaces/CarTrackSnapshotTest.kt

```diff
@@ -0,0 +1,52 @@
+package camp.nextstep.edu.racingcar.interfaces
+
+import camp.nextstep.edu.racingcar.racing.Car
+import camp.nextstep.edu.racingcar.racing.CarRaceEvent
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldContain
+import io.kotest.matchers.string.shouldNotContain
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+internal class CarTrackSnapshotTest {
+
+    @DisplayName("자동차가 움직인 적 없는 경우 ⎼ 는 없다.")
+    @Test
+    fun shouldReturnOnlyCarName() {
+        val carName = "tim"
+
+        val events = listOf(
+            CarRaceEvent(0, 0, false),
+            CarRaceEvent(1, 0, false),
+            CarRaceEvent(2, 0, false),
+            CarRaceEvent(3, 0, false),
+            CarRaceEvent(4, 0, false),
+            CarRaceEvent(5, 0, false),
+        )
+        val carTraces = CarTrackSnapshot(Car(0, carName), events).toString()
+
+        carTraces.shouldContain(carName)
+        carTraces.shouldNotContain("⎼")
+    }
+
+    @DisplayName("자동차가 5번 움직인 경우 ⎼ 는 5개 이다.")
+    @Test
+    fun shouldReturnsTwoDash() {
+        val carName = "tim"
+
+        val events = listOf(
+            CarRaceEvent(0, 0, true),
+            CarRaceEvent(1, 0, true),
+            CarRaceEvent(2, 0, true),
+            CarRaceEvent(3, 0, false),
+            CarRaceEvent(4, 0, true),
+            CarRaceEvent(5, 0, true),
+            CarRaceEvent(6, 0, false),
+        )
+        val carTraces = CarTrackSnapshot(Car(0, carName), events).toString()
+
+        carTraces.shouldContain(carName)
+        val dashCount = carTraces.count { it == '⎼' }
+        dashCount.shouldBe(5)
```

Comment:
```suggestion
        val carTraces = CarTrackSnapshot(Car(0, "tim"), events).toString()
        assertThat(carTraces).isEqualTo("tim\t: ⎼ ⎼ ⎼ ⎼ ⎼  ")
```
저는 이렇게 검증하는게 더 좋다고 생각합니다. 이유는 테스트를 읽기 쉽게 만들 수 있으며 결국 유지보수 때문이에요.

또다른 의견 하나는 junit 프레임워크에서 kotest 단정문을 혼용하고 있는데요.. 이런 경우를 본 적이 없어서 약간 의견내긴 조심스럽긴 하지만 바로 든 생각은 kotest 와 junit 은 각각 별개의 테스트 프레임워크이니 혼용하면 예상치 못한 예외가 있지 않을까 합니다.

Reply:
junit 이 익숙한데,, kotest 는 써보고 싶고...
그렇게 혼종이 되었네요.. 😂